doctrine.rst

```diff
@@ -746,6 +746,23 @@ In the example above, the ``$product`` argument is handled automatically,
 but ``$comment`` is configured with the attribute since they cannot both follow
 the default convention.
 
+If you need to get other information from the request to fetch your entity, you
+can also access to the request in your expression thanks to the ``request``
+variable. Let's say you pass the page limit of a list in a query parameter::
+
+    #[Route('/product/{id}/comments')]
+    public function show(
+        Product $product,
+        #[MapEntity(expr: 'repository.findBy(["product_id" => id], null, request.query.get("limit", 10)')]
+        Comment $comment
```

Comment:
shouldn'it be an array/collection ? due to the `findBy`

Reply:
That's right, thanks!